### ISSUE TITLE
No support

### ISSUE THREAD

**Issue by nuddik** [nylas/nylas-mail#3546]
It's pretty sad, but I cancelled my pro subscription today as there are just too many problems and no response to them :( 

I'll check back in 6 months and see how it's going, I really like the email client, but it just doesn't quite work.

Problems I had:

Slow sync
Incomplete search
delay of about 10 seconds when changing between accounts
Other little annoyances I could have lived with if the above worked

I really hope this project works and I can come back and use it in the future and pay, but for now I just can't justify paying for an email client I can't use and where issues are not responded to.

Good luck, hopefully I'll be back in the future :(

**Comment by shenanigans12** [comment]
The number of issues keeps climbing. Their servers went offline today and prevented sending and receiving mail, checking for updates, and loading pages on their site for about 2 hrs.  

I just tried to switch to Nylas Mail from N1 and the signature issue that prevented me from moving over in FEBRUARY is still present (35 releases later).   It looks like a sinking ship. I think I'll be switching to something like Spark or Polymail that is actually stable and supported.

**Comment by alberto2000** [comment]
Same here, I'll have to cancel my subscription also because they dropped Exchange support for whatever reason. Will be checking back another time.

**Comment by reyman** [comment]
Same here, i'm a pro user.
We need more support from your teams, or at least more communication.

**Comment by shrinkray** [comment]
Agreed. My sync stopped after an SSD upgrade; the cause of the app failure is still unknown. Possibly something unrelated. My subscription is due to renew soon, so it is good to know I wasn't alone.
